install/relay-credentials.sh

```diff
@@ -20,6 +21,10 @@ if [[ ! -f "$RELAY_CREDENTIALS_JSON" ]]; then
     > "$RELAY_CREDENTIALS_JSON"
```

Comment:
Maybe we should add that `-T` here right after `$dcr`?

Reply:
Because, read the giant comment I left there? 😁

Reply:
I read it but I couldn't make sense of it.

> `relay generate credentials` tries to read the config and the credentials even with the `--stdout` and `--overwrite` flags and then errors out when the credentials file exists but not valid JSON. We hit this case as we redirect output to the same config folder, creating an empty credentials file before relay runs.

So no shell redirect means no credentials file created before relay runs means no error parsing JSON?